torch/csrc/jit/profiling_graph_executor_impl.cpp

```diff
@@ -161,6 +161,7 @@ ExecutionPlan ProfilingGraphExecutorImpl::getPlanFor(Stack& stack) {
   runProfilingOptimizations(copy);
   // cache
   optimized_plan_ = ExecutionPlan(copy);
+  optimized_plan_->code.setNumBailOuts(num_bailouts_);
```

Comment:
Can the number of bailuts be passed as an argument to ExecutionPlan? It is weird for it to be a setter called right after a constructor.